it/uk/gov/hmrc/soletraderidentificationfrontend/utils/ViewSpecHelper.scala

```diff
@@ -52,7 +52,7 @@ object ViewSpecHelper {
 
     lazy val getSubmitButton: Elements = element.getElementsByClass("govuk-button")
 
-    lazy val getHintText: String = element.select(s"""span[class=govuk-hint]""").text()
+    lazy val getHintText: String = element.select("#postcode-hint > p").text()
```

Comment:
Potentially re name this postcode specific, as it only finds the postcode hint. Or use generic hint find: element.getElementsByClass("govuk-hint").text()

Reply:
done